packages/core/src/tools/ContactProvider.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Platform} from 'react-native';
+import Toast from 'react-native-toast-message';
+import {PERMISSIONS, Permission, request} from 'react-native-permissions';
+import * as Contacts from 'react-native-contacts';
+
+interface ContactData {
+  firstName: string;
+  lastName?: string;
+  phoneNumbers?: {
+    mobilePhone: string;
+    fixedPhone: string;
+  };
```

Comment:
Both phone numbers are not required. I think it should be better to just ask for mobilePhone and fixedPhone directly instead of wrapping them by phoneNumbers

```
interface ContactData {
  firstName: string;
  lastName?: string;
  mobilePhone?: string;
  fixedPhone?: string;
  email?: string;
  address?: string;
  notes?: string;
```